app/scripts/modules/amazon/src/loadBalancer/AmazonLoadBalancersTag.tsx

```diff
@@ -40,15 +44,15 @@ class LoadBalancerListItem extends React.Component<ILoadBalancerListItemProps> {
 }
 
 @autoBindMethods
-class LoadBalancerButton extends React.Component<ILoadBalancerListItemProps> {
+class LoadBalancerButton extends React.Component<ILoadBalancerSingleItemProps> {
   private onClick(e: React.MouseEvent<HTMLElement>): void {
     this.props.onItemClick(this.props.loadBalancer);
     e.nativeEvent.preventDefault(); // yay angular JQueryEvent still listening to the click event...
   }
 
   public render(): React.ReactElement<LoadBalancerButton> {
     return (
-      <Tooltip value={`Load Balancer: ${this.props.loadBalancer.name}`}>
+      <Tooltip value={`${this.props.label || 'Load Balancer'}: ${this.props.loadBalancer.name}`}>
```

Comment:
seems like the label and the default are the same - should this be `${this.props.label || 'Load Balancers'`?